vector/src/main/java/im/vector/app/features/location/LocationSharingAction.kt

```diff
@@ -19,5 +19,8 @@ package im.vector.app.features.location
 import im.vector.app.core.platform.VectorViewModelAction
 
 sealed class LocationSharingAction : VectorViewModelAction {
-    object OnShareLocation : LocationSharingAction()
+    object CurrentUserLocationSharingAction : LocationSharingAction()
+    data class PinnedLocationSharingAction(val locationData: LocationData?) : LocationSharingAction()
+    data class LocationTargetChangeAction(val locationData: LocationData) : LocationSharingAction()
+    object ZoomToUserLocationAction : LocationSharingAction()
```

Comment:
Can you remove Action suffix like `ZoomToUserLocation` to continue the naming convention of other actions?